src/extension.ts

```diff
@@ -0,0 +1,40 @@
+'use strict';
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import {spawnSync} from 'child_process';
+
+export function activate(context: vscode.ExtensionContext) {
+  console.log('Activated!');
+  vscode.languages.registerDocumentFormattingEditProvider('serenity-gml', {
+    provideDocumentFormattingEdits(document: vscode.TextDocument) {
+      // FIXME: Find a way how we can avoid forcing a save
```

Comment:
It would probably be best if we could pipe `gml-format` some stdin and get the formatted stuff on stdout. That would also allow for partial-file formatting.

Reply:
Made that two FIXMEs